### ISSUE TITLE
Allows the API users to change existing scenario's?

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etengine#1043]
When you open a link to a scenario, the API creates a new scenario with unique ID. This prevents users from messing up existing scenario's.

The API allows a user to specify an ID and change sliders right? Is it therefore possible to mess with existing (saved) scenario's of other users?

**Comment by antw** [comment]
> The API allows a user to specify an ID and change sliders right?

Yes, it is possible to do this through the API. You cannot change the area code or end year, but slider settings may be updated.

If we want to be paranoid (perhaps we should?), we could treat "protected" scenarios as immutable and deny any further changes through the API.

**Comment by ChaelKruip** [comment]
> If we want to be paranoid (perhaps we should?), we could treat "protected" scenarios as immutable and deny any further changes through the API.

I think we should. Is this easy to do?

**Comment by antw** [comment]
> Is this easy to do?

Reasonably. There are a few places that would be affected in ETE, but it can certainly be done.
